### PR TITLE
Add simple comment system

### PR DESCRIPTION
**Pull request by alessandroHenrique**
How to test:
- Install the new dependencies in the requirements.txt file
- Enter in any error page without authentication, you should not be able to see the comment form
- Enter in any error page with authentication, you should be able to see the comment form
- Try submit a comment with just plain text or with code

Comment section right beneath the How to reproduce and How to fix section:
comment_section

Comment form right beneath the Comment section:
comment_form

### CODE REVIEW

**Comment by alessandroHenrique** [comment]
Work in progress. Finishing some details

File: error_posts/forms.py [hunk 1]
```diff
@@ -38,3 +41,10 @@ def clean_recaptcha(self):
             if not res['success']:
                 raise forms.ValidationError('Invalid Recaptcha')
         return code
+
+
+class CommentFormWithMarkDown(CommentForm):
+
+    def __init__(self, *args, **kwargs):
+        super(CommentFormWithMarkDown, self).__init__(*args, **kwargs)
+        self.fields['comment'].widget = MarkdownWidget()
```
Comment (andersonresende) [inline error_posts/forms.py]: I think you can do this without override the ` __init__`  method. Like this 
`comment = forms.CharField(widget=MarkdownWidget())`

File: templates/error_posts/detail.html [hunk 2]
```diff
@@ -46,20 +50,14 @@ <h2 class="panel-title">How to fix</h2>
...
+    {% render_comment_list for object %}
+  </div>
+  {% if user.is_authenticated %}
+    <div class="col-md-12">
+      {% render_comment_form for object %}
+    </div>
+  {% endif %}
 </div>
 
-<!-- Disqus -->
```
Comment (andersonresende) [inline templates/error_posts/detail.html]: Have you searched for `disqus` or `DISQUS` on the project? Maybe could exist  some references in other places of the code.

File: error_posts/forms.py (hunk 1 above)
Comment (alessandroHenrique) [inline error_posts/forms.py]: Oh, good idea. It works!

File: templates/error_posts/detail.html (hunk 2 above)
Comment (alessandroHenrique) [inline templates/error_posts/detail.html]: Yes, I did. Didn't find anything, I even tried searching for `disq`

**Comment by renatooliveira** [comment]
Looks awesome @alessandroHenrique. Two small fixes
- When there is no comment, we don't need to show the `Comments` section empty
- When we have just one comment, the line between comments could be hiden too

**Comment by alessandroHenrique** [comment]
@renatooliveira All done. Please take another look at the comment section

**Comment by renatooliveira** [comment]
Perfect, thanks!

**Pull request merged**